sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -928,8 +928,8 @@ def reset(self):
     for op in self.ops.values():
       op.reset()
 
-  def process_bundle(self, instruction_id):
-    # type: (str) -> Tuple[List[beam_fn_api_pb2.DelayedBundleApplication], bool]
+  def process_bundle(self, instruction_id, use_task_worker=True):
+    # type: (str, bool) -> Tuple[List[beam_fn_api_pb2.DelayedBundleApplication], bool]
```

Comment:
@violalyu Is the `BundleProcessor` re-used for many bundles?   If not, should `use_task_worker` be passed to `__init__`?

Reply:
IIUC, a `BundleProcessor` may be reused, the number of bundle processor should be the same as the number of SDK worker we have I think (each SDK worker request `BundleProcessor` from `BundleProcessorCache`, and if there's no more "available" bundle processor it creates a new one, otherwise it use the available ones)

Reply:
Can `use_task_worker` vary over the lifetime of a `BundleProcessor`?  If not, we should make it an `__init__` arg.

Reply:
I updated in this commit: 884ffd332a342760bb50c2b64c135dabd300d4f7